provider/go.mod

```diff
@@ -11,8 +11,8 @@ require (
 	github.com/onsi/ginkgo v1.12.0 // indirect
 	github.com/onsi/gomega v1.9.0 // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/pulumi/pulumi/pkg/v3 v3.2.0
-	github.com/pulumi/pulumi/sdk/v3 v3.2.0
+	github.com/pulumi/pulumi/pkg/v3 v3.2.2-0.20210512083603-840c7a5a13c1
```

Comment:
Perhaps 2288f8b7fb97fc652e5ca3afdd623dedc3b92ef5 here to be pedantic (post addressing PR feedback and merging to master).

Reply:
I guess I grabbed the commit right after, but it doesn't contain any further codegen changes, so I'll leave this in the interest of getting a release out now.

Reply:
Ah if it's after we're good.